satpy/composites/__init__.py

```diff
@@ -1014,19 +1014,22 @@ class CloudCompositor(GenericCompositor):
     """Detect clouds based on thresholding and use it as a mask for compositing."""
 
     def __init__(self, name, transition_min=258.15, transition_max=298.15,  # noqa: D417
-                 transition_gamma=3.0, **kwargs):
+                 invert_alpha=False, transition_gamma=3.0, **kwargs):
```

Comment:
for backwards compatibility, `invert_alpha` should be placed after `transition_gamma`

Reply:
Fixed now.